packages/aws-cdk/lib/api/bootstrap/bootstrap-environment.ts

```diff
@@ -170,7 +171,7 @@ export class Bootstrapper {
       case 'custom':
         return loadStructuredFile(this.source.templateFile);
       case 'default':
-        return loadStructuredFile(path.join(__dirname, 'bootstrap-template.yaml'));
+        return loadStructuredFile(path.join(rootDir(), 'lib', 'api', 'bootstrap', 'bootstrap-template.yaml'));
```

Comment:
Alternatively - code-gen a `.ts` file with a constant string that holds the data...

Reply:
Not in favor of adding a codegen step for this. This should be ok for now.